.idea/ClojureProjectResolveSettings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
remove this file

Reply:
@divs1210 this should be in gitignore, putting it in gitignore

Reply:
@kumar88kanishk also remove this file. it should not be committed.